crates/python/Cargo.toml

```diff
@@ -33,6 +33,11 @@ rigetti-pyo3.workspace = true
 paste = "1.0.11"
 pyo3-log = "0.8.2"
 once_cell = "1.18.0"
+pyo3-opentelemetry = { version = "0.2.0-rc.0" } 
+opentelemetry = "0.20.0"
+tracing = "0.1.37"
+pyo3-tracing-subscriber = { version = "0.2.0-rc.0", features = ["layer-otel-otlp-file", "layer-otel-otlp"] }
```

Comment:
I'm noticing we missed this with `pyo3-log`, but can you specify these in the root `Cargo.toml` with the other `pyo3` dependencies? They often need to be updated together, so grouping them there makes that process a bit easier.